README.md

```diff
@@ -5,3 +5,158 @@
 - Web and Network  
 - Software Development
 - Web Development
+
+### BASIC COMPUTER TERMS
+
+#### Computer
+A computer is a device that executes a program or programs. 
+
+#### Hardware:
+Hardware consists of the physical components of your computer. If you can kick it, punch it, or pick it up to throw it, it’s hardware. For e.g. Mouse, Keyboard, Monitor are hardware
+
+#### Software	
+A software is a program application; which provides the computer with step-by-step instructions to perform a specific task. A computer cannot function without software.
+
+#### Application Program
+A Program that has been created to perform a specific task that is useful to the user. Examples include Word Processors (Word), Spreadsheets (Excel), Browsers (Netscape) and Drawing Packages (Paint).
+
+#### CPU 
+The central processing unit or CPU, also known as the microprocessor or processor. The CPU performs basic arithmetic, logic, controlling, and input/output (I/O) operations specified by the instructions
+
+#### Operating System  
+The operating system (OS) of a computer is the software that communicates with the computer hardware. No software can run on the computer without the operating system.
+
+#### File 
+A file is a block of information that includes word documents, pictures, music, etc. Each file is differentiated with the help of a unique file name. 
```

Comment:
Each file has a extension that define it's type. Images can be denoted as JPEG, PNG, GIFs. Text files can be TXT, DOCX or PDF, music or sounds can be MP3 and videos are mostly MP4